pkg/controller/plan/adapter/ovirt/builder.go

```diff
@@ -88,7 +88,8 @@ var osMap = map[string]string{
 	"rhel_7_ppc64":         "rhel7.7",
 	"rhel_7_s390x":         "rhel7.7",
 	"rhel_7x64":            "rhel7.7",
-	"rhel_8x64":            "rhel8.1",
+	"rhel_8x64":            "rhel8.9",
+	"rhel_9x64":            "rhel9.3",
```

Comment:
@liranr23 did you make sure the osinfo version used by kubevirt already supports these versions? they are fairly new

Reply:
to me it looked as they just pointing to the libosinfo

Reply:
yeah but the question is whether all the supported versions of kubevirt and openshift virtualization use a version of libosinfo that already has support for, e.g., rhel 8.9

Reply:
or maybe I don't understand your previous message - you mean that they handle both rhel 8.9 and 8.1 the same way?

Reply:
in 4.13 they have 8.8 and 9.2.. so i change to that...